### DISCUSSION TITLE
qemu-img convert file driver requires XXX to be a regular file

### DISCUSSION THREAD

**Discussion by charlotteKo** [Homebrew/discussions#4528] [Everyday usage] [answered]
### Output of `brew config`

```shell
qemu-img convert -O vmdk /Users/charlotteendo/Desktop/Windows11_InsiderPreview_Client_ARM64_en-us_22598.VHDX ~ /Desktop/Install/Windows11ARM.vmdk

qemu-img: Could not open '/Users/charlotteendo': 'file' driver requires '/Users/charlotteendo' to be a regular file
```

### Description of issue

I am trying to install Windows11 on a Mac with VM-Ware (https://www.youtube.com/watch?v=Pt5FIO_I-qo).  Therefore I am using Homebrew to convert the downloaded Windows file (Windows11_InsiderPreview_Client_ARM64_en-us_22598.VHDX) into VMDW using qemu-img.

The tutorial I am watching tells me to wirte: 

qemu-img convert -O vmdk /Users/charlotteendo/Desktop/Windows11_InsiderPreview_Client_ARM64_en-us_22598.VHDX ~ /Desktop/Install/Windows11ARM.vmdk

which gets me the following:
qemu-img: Could not open '/Users/charlotteendo': 'file' driver requires '/Users/charlotteendo' to be a regular file

Can anyone help me with that?

**Comment by ZhongRuoyu** [comment, marked as answer]
Is there an extra space after the tilde `~` in your command? It seems to me that this should be:

```
qemu-img convert -O vmdk /Users/charlotteendo/Desktop/Windows11_InsiderPreview_Client_ARM64_en-us_22598.VHDX ~/Desktop/Install/Windows11ARM.vmdk
```

**Reply by charlotteKo** [reply]
Thank you very very much, that was it!

**Comment by gromgit** [comment]
I don't know which tutorial you're following, but the error message makes me 100% sure that you typed the command wrong:
>```
>qemu-img ...22598.VHDX ~ /Desktop/Install/Windows11ARM.vmdk
>                         ^ this space should not exist
>```
